environment-properties/intg_properties.json

```diff
@@ -17,8 +17,8 @@
         "http://localhost:9000/*"
       ],
       "webOrigins": [
-        "https://tdr-integration.nationalarchives.gov.uk/*",
-        "http://localhost:9000/*"
+        "https://tdr-integration.nationalarchives.gov.uk",
+        "http://localhost:9000"
```

Comment:
Why is this change necessary? Could you add an explanation to the commit message?

Reply:
In existing intg Keycloak configuration values were set without '/*'

Having '/*' appended to the web origin was causing the tdr-fe login at upload to fail

Reply:
Something strange has happened to the commit history. Is it possible to push it so it's just a single commit with the updated commit message? Let me know if you want a hand with git wrangling.

(I realise I'm being picky, but I've been digging around in the git history recently trying to understand why something changed, and it's reminded me how useful it is if the history is meaningful.)